docs/source/examples/workspaces/dummy/PinFile

```diff
@@ -30,8 +30,7 @@ dummy-new:
         all:
           vars:
             ansible_user: root
-            ansible_private_key_file: |
-              "{% raw -%}{{ lookup('env', 'TESTLP') | default('/tmp', true) }}/CSS/keystore/css-central{%- endraw %}"
+            ansible_private_key_file: "{% raw -%}{{ lookup('env', 'TESTLP') | default('/tmp', true) }}/CSS/keystore/css-central{%- endraw %}"
```

Comment:
@p3ck This was the problem.  With the template on a separate line, the string was treated as if it did not have any quotes.  Single quotes were added and the ones in the middle of the string were escaped.  These then became '\\\' while the templates were being rendered, hence the strange error